rules.js

```diff
@@ -13,11 +13,19 @@ var _ = require('lodash');
 module.exports = function(styles, opts={}) {
   const enableLightBox = opts.enableLightBox || false
   const navigator = opts.navigator
+
+  const LINK_INSIDE = "(?:\\[[^\\]]*\\]|[^\\]]|\\](?=[^\\[]*\\]))*";
+  const LINK_HREF_AND_TITLE =
+          "\\s*<?([^\\s]*?)>?(?:\\s+['\"]([\\s\\S]*?)['\"])?\\s*";
   return {
     autolink: {
       react: function(node, output, state) {
         state.withinText = true;
-        var pressHandler = function() { Linking.openURL(node.target) };
+        var pressHandler = function() {
+          if(opts.onLink) {
+            opts.onLink(node.target)
```

Comment:
On second thoughts, can we refactor this to a top level function if it's repeated so many times?